docs/source/guides/data-scientist/01-search-for-datasets.rst

```diff
@@ -0,0 +1,159 @@
+
+:: 
+   
+   pip install -U hagrid
+   hagrid quickstart data-scientist
+
+
+
+In the last tutorial, you learned :doc:`How to Connect to a Domain Server <00-deploy-domain>`
+that allows us to connect to your organization’s private data servers.
+
+Once we are connected to the data servers, the first thing that we 
+would like to do is to look for the available datasets on it. This 
+is exactly what we are going to cover in this tutorial.
+
+After today’s tutorial, you will learn how to ``search for datasets`` 
+on the ``domain node`` you are connected to. 
+
+   **Note:** Throughout the tutorials, we mean Domain Servers
+   whenever we refer to Domain Node. Both point to the same and are used
+   interchangeably.
+
+Steps to Search for Datasets on a Domain
+---------------------------
+
+📒 Overview of this tutorial:  
+
+#. **Login** to the Domain 
+#. **List** the Datasets on the Domain 
+#. **Choose** a Dataset
+#. **Preview** the Description of the chosen Dataset
+
+|01-upload-data-00|
+
+Step 1: Import Syft
+~~~~~~~~~~~~~~~~~~~
+
+To utilize the privacy-enhancing features offered in PyGrid and to 
+communicate with your domain node, you must first ``import`` OpenMined's 
+``private`` deep learning library: PySyft.
+
+Let's import Syft by running the below cell:
+
+::
+
+   In:
+
+   # run this cell
+    try:
+        import syft as sy
+        print("Syft is imported")
+    except:
+        print("Syft is not installed. Please use the 🧙🏽‍♂️ Install Wizard above.")
+
+   Out: Syft is imported
+
+Step 2: Log into Domain
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's login to our Domain with the credentials provided to you by a domain owner. 
+If you do not have a domain owner, you can create one locally for yourself following 
+the tutorials starting here: `data-owner/00-deploy-domain <../data-owner/00-deploy-domain.html>`_.
+
+To login to your Domain node, you will need to define which Domain you are logging into and who you are. 
+In this case, it will take the form of:
+
+* IP Address and Port of the domain host
+* Your user account Email and Password
+
+.. warning::
+    Make sure to use the Data Scientist credentials provided to you.
+
+::
+
+   In:
+
+    try:
+        domain_client = sy.login(
+        port=8081,
+        email="jane@email.com",
+        password="supersecurepassword"
+        )
+    except Exception as e:
+        print("Unable to login. Please check your crendentials and the domain with your domain owner")
+
+   Out:
+    Connecting to <IP-Address>... done! Logging into <domain-name>... done!
+
+Amazing :) You have just logged in to your Domain and have a domain client with us to explore further.
+
+Step 3: Search for Datasets on the Domain
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Now that we have an authenticated domain client with 
+us, we will look out for the datasets available 
+on this domain with the following command:
+
+::
+
+   In:
+   domain_client.datasets
+
```

Comment:
Can we add an example image of the output shown from domain_client.datasets, where the Age dataset is present at zero index.